src/02-common-principles.md

```diff
@@ -364,14 +364,19 @@ is NOT RECOMMENDED to maintain their uniqueness.
 
 ## Units
 
-All units SHOULD be specified as per International System of Units (abbreviated
+All units SHOULD be specified as per [International System of Units](https://en.wikipedia.org/wiki/International_System_of_Units)
+(abbreviated
 as SI, from the French Système international (d'unités)) and can be SI units or
-SI derived units. In case there are valid reasons to deviate from SI units or SI
+[SI derived units](https://en.wikipedia.org/wiki/SI_derived_unit).
+In case there are valid reasons to deviate from SI units or SI
 derived units, the units MUST be specified in the sidecar JSON file. In case
 data is expressed in SI units or SI derived units, the units MAY be specified in
-the sidecar JSON file. In case prefixes are added to SI or non-SI units (e.g.,
-mm), the prefixed units MUST be specified in the JSON file (see [Appendix V](99-appendices/05-units.md):
-Units). In particular:
+the sidecar JSON file. In case non-standard prefixes are added to SI or non-SI
+units, these non-standard prefixed units MUST be specified in the JSON file.
```

Comment:
just a quick clarification.  does this mean that every bids dataset must have a `participants.json` which lists the unit of `Age` as `years`?

Reply:
this would be great from my perspective, but just want to verify that this is the intent.

Reply:
`participants.json` MUST be present, but the column `age` is optional according to https://bids-specification.readthedocs.io/en/stable/03-modality-agnostic-files.html#participants-file. But although common for age, "year" is actually not an SI unit - and not a very accurate unit either (every 4 years it is off by 1/365). 

However, I don't think that the proposed changes would change the specification in this respect. The spec does (still) state "Age SHOULD be given as the number of years since birth at the time of scanning ..." allowing for deviations, e.g. when recording data from babies or toddlers (whose age is more conveniently quantified in months).